packages/client/mutations/UpgradeToTeamTierMutation.ts

```diff
@@ -37,10 +36,12 @@ graphql`
 `
 
 const mutation = graphql`
-  mutation UpgradeToTeamTierMutation($orgId: ID!, $stripeToken: ID, $paymentMethodId: ID) {
-    upgradeToTeamTier(orgId: $orgId, stripeToken: $stripeToken, paymentMethodId: $paymentMethodId) {
-      error {
-        message
+  mutation UpgradeToTeamTierMutation($invoiceId: ID!) {
```

Comment:
`upgradeToTeamTier` is now on the private schema, so we don't need a client mutation, but we do need the fragments for the subscription. Perhaps the fragments could be created elsewhere instead of here so we don't have this mutation 🤷

Reply:
We have a fragment folder, so I've moved this there 👍